No issue - Little improvements (#45)
DANS-KNAW/easy-sword2-dans-examples: README.md

Getting started
---------------

> The following is a step-by-step instruction on how to run a simple example using the DANS acceptance test server at https://demo.easy.dans.knaw.nl. 

Reviewer: ```suggestion
The following is a step-by-step instruction on how to run a simple example using the DANS acceptance test server at `https://demo.easy.dans.knaw.nl`. 
```

> The following is a step-by-step instruction on how to run a simple example using the DANS acceptance test server at https://demo.easy.dans.knaw.nl/home. 

Reviewer 2: this url does not exist. I'd leave it at
```suggestion
The following is a step-by-step instruction on how to run a simple example using the DANS acceptance test server at https://demo.easy.dans.knaw.nl/ 
```

* `<prefix>-doi-report-<date>.csv`: An overview of all the doi with this prefix in the DANS archives.

`DANS-EASY Error report: status of failed EASY deposits` this e-mail contains two reports about failed deposits: 
* `DANS-EASY-report-error-yesterday-<date>.csv`: A deposit-report with all the FAILED / REJECTED / INVALID deposits of the last day. 
* `DANS-EASY-report-error-<date>.csv`: A deposit-report with all the failed deposits that are in the deposit area. In case a `REJECTED` deposit has been resend, the old one is still mentioned here.

> `DANS-EASY Report: status of EASY deposits`, A report email with all deposits in the deposit area:

Reviewer 2: ```suggestion
`DANS-EASY Report: status of EASY deposits`, An e-mail with reports on all deposits in the deposit area:
```
